packages/edit-site/src/components/editor/index.js

```diff
@@ -29,21 +30,23 @@ function Editor( { settings } ) {
 		[]
 	);
 	return template ? (
-		<SlotFillProvider>
-			<DropZoneProvider>
-				<EntityProvider
-					kind="postType"
-					type="wp_template"
-					id={ settings.templateId }
-				>
-					<Notices />
-					<Header />
-					<Sidebar />
-					<BlockEditor settings={ settings } />
-					<Popover.Slot />
-				</EntityProvider>
-			</DropZoneProvider>
-		</SlotFillProvider>
+		<GlobalStylesStateProvider>
```

Comment:
Does Global Style need to be the outer-most provider?

Reply:
For this current implementation. Most likely would not once content is provided by `@wordpress/data`. At the moment, I'm simulating that data flow with a `React.Context` instance